roles/setup-local/tasks/main.yml

```diff
@@ -38,4 +36,4 @@
 - name: Update drush aliases
   template:
     src: "acsf.aliases.drushrc.php"
-    dest: "{{ localhost_home }}/.drush/acsf.aliases.drushrc.php"
+    dest: "~/.drush/acsf.aliases.drushrc.php"
```

Comment:
I'm a little concerned about clobbering files in a user's home directory here. Should we make a backup first? Or should we just declare that the `acsf.aliases.drushrc.php` namespace belongs to this tool?

Reply:
Since it's probably just going to be you, me, and perhaps Mike, we could also remove this local setup role.  The reason I like Ansible managing it is because that file changes, or at least had to change several times in development, and it was helpful to only have to update it in one place (and not exchange gists or files on Slack).